### PR TITLE
fix: Disable transaction buttons until sdk is initialized

### LINKED ISSUE #4404: Error when trying to sign/execute transaction too fast after page refresh
## Bug description

When loading the app and pressing the Confirm button in the queue there is sometimes an error that the SDK could not be initialized. The SDK takes a bit until ready but if a user tries to "interact" with it too fast it will error out. This especially happens with e2e tests that do this step programmatically and run into this error more often which makes tests more flaky.

## Steps to reproduce
 1. Add a transaction to the queue
 2. Go to the queue
 3. Reload the app
 4. Press the confirm button right after reloading
 5. Observe an error

## Expected result

There should not be an error

## Obtained result

There is an error visible in the UI

### PR DESCRIPTION
**Pull request by usame-algan**
## What it solves

Resolves #4404

This should make our e2e tests more robust

## How this PR fixes it

- Adds a check for the `sdk` inside `CheckWallet` which will disable all buttons it wraps until the SDK is initialized

## How to test it

1. Create and queue a transaction
2. Go to the queue
3. Reload the app
4. Observe the Confirm button is disabled until the SDK is ready

## Checklist
* [ ] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

**Comment by usame-algan** [comment]
@mike10ca The e2e tests can be tested in this PR. Lets see if they still fail as often as before. Do we have a list of e2e test where we know this error happens?

File: src/components/common/CheckWallet/index.tsx [hunk 1]
```diff
@@ -46,6 +49,9 @@ const CheckWallet = ({
     if (!wallet) {
       return Message.WalletNotConnected
     }
+    if (!sdk) {
+      return Message.SDKNotInitialized
```
[the remark this answers is not in the document]
Comment (usame-algan) [reply src/components/common/CheckWallet/index.tsx:53]: We had a mix of different selectors in those tests so I also refactored them to use `getByText` and `getByLabelText` for consistency.

**Review by schmanu — APPROVED** [review]

**Comment by mike10ca** [comment]
> @mike10ca The e2e tests can be tested in this PR. Lets see if they still fail as often as before. Do we have a list of e2e test where we know this error happens?

@usame-algan yes, this was associated with sending out an NFT. I tested it locally few times and it seems to work. Let's merge it and observe the tests further.

**Review by mike10ca — APPROVED** [review]

**Comment by usame-algan** [comment]
@francovenica I suggest we test the app with different safe versions. Basically we disable a lot of buttons now if the SDK could not be initialized so we need to make sure it still works even with older safe versions where the SDK might not work.

**Comment by francovenica** [comment]
So for me it works fine, and shows the tooltip about the sdk not being enabled.
image

I tried in old safes versions, safes 1.1.1 and 1.3.0 and I also there I got the button being disabled for a second just with no tooltip.

Since it works fine, let's see if automation works fine now since it is the reason we are doing this mostly

**Pull request merged**